storage/remote/client.go

```diff
@@ -46,6 +47,10 @@ const maxErrMsgLen = 1024
 
 var UserAgent = fmt.Sprintf("Prometheus/%s", version.Version)
 
+var ErrStatusBadRequest = errors.New("HTTP StatusBadRequest") // 400
+
+var ErrStatusNotAcceptable = errors.New("HTTP StatusNotAcceptable") // 406
```

Comment:
It would be epic to add quick comment on both that explains semantics of those?

Reply:
Comments added.